### DISCUSSION TITLE
4주차 과제 - (Poster) Design With Code ver 2.0

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#14] [Show and tell]
3주차 과제 수정 및 발전

* 1개 이상의 변수 값을 (if 구문으로) 제어하여 반복되는(루핑) 애니메이션 추가

* if 구문을 활용한 인터랙션 추가 - 마우스, 키보드

+ 1개 이상의 스크린샷 이미지
+ (모션이나 인터랙션 있다면) 스크린 동영상 캡쳐하여 유튜브 업로드 유튜브 embed

+ 간단한 설명 (왜 / 어떻게)
+ p5js editor 의 File - Share - Present 링크
+ p5js editor 의 File - Share - Edit 링크
+ 코드 복사 붙여넣기

**Comment by eunjinhong** [comment]
홍은진
https://blog.naver.com/tnekf1012/222533757461

**Comment by imyeeunok** [comment]
김예은
https://blog.naver.com/010312kye/222533829921

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/khj317888/222533831455

**Comment by Joeun410** [comment]
김조은
https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222533920277&categoryNo=11&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=38829

**Comment by jwejwepark** [comment]
박주혜
https://blog.naver.com/glqkfl0726/222533932750

**Comment by SeyounJung01** [comment]
정세윤
https://blog.naver.com/seyounjj2001/222534045800
